source/reference/mongodb-wire-protocol.txt

```diff
@@ -117,6 +117,14 @@ follows:
 Request Opcodes
 ~~~~~~~~~~~~~~~
 
+.. note:: 
+   Starting with MongoDB 2.6 and :data:`~isMaster.maxWireVersion` ``3``, 
+   modern drivers use the :ref:`database commands<collection-commands>` 
```

Comment:
I would assume drivers that support MongoDB 2.6+. I think maxWireVersion should generally be at least MongoDB 3, right? So this might be better written as 'drivers supporting MongoDB 2.6+'

@ajdavis , does that sound about right?